lib/odbc_adapter/adapters/null_odbc_adapter.rb

```diff
@@ -4,15 +4,13 @@ module Adapters
     # registry. This allows for minimal support for DBMSs for which we don't
     # have an explicit adapter.
     class NullODBCAdapter < ActiveRecord::ConnectionAdapters::ODBCAdapter
-      class BindSubstitution < Arel::Visitors::ToSql
-        include Arel::Visitors::BindVisitor
-      end
-
       # Using a BindVisitor so that the SQL string gets substituted before it is
       # sent to the DBMS (to attempt to get as much coverage as possible for
       # DBMSs we don't support).
       def arel_visitor
-        BindSubstitution.new(self)
+        # Using PostgresSQL visitor since BindVisitor is deprecated in Arel 9.0
+        # We only need Postgre Adapter.
+        Arel::Visitors::PostgreSQL.new(self)
```

Comment:
Let's see if we can run the specs at least once to make sure that they only fail on MySQL-specific specs. If not, we might've broken more than we intended to.